utensor_cgen/cli.py

```diff
@@ -104,13 +104,9 @@ def show_graph(model_file, **kwargs):
 
 def _show_pb_file(pb_file, **kwargs):
```

Comment:
Cant torch graphs also be represented in protobuf formats? This naming might get confusing

Reply:
Which naming?
`_show_pb_file`?

FYI, yes, a torch graph can be represented in protobuf format.
First you save torch graph in `onnx` format and convert `onnx` to `GraphDef` via `onnx-tf`.

Reply:
Cool, let's go ahead and leave it as is, but make it a TODO in the issue tracker.

Reply:
I know.
I promise.